style.css

```diff
@@ -1,3 +1,192 @@
+*{
```

Comment:
All I have to say about your CSS is that it looks super neat, tidy and easy to read.
Well done! 😁